onnxruntime/core/providers/webnn/builders/helper.h

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <core/common/status.h>
+#include "core/common/inlined_containers.h"
+#include <core/graph/basic_types.h>
+#include "core/providers/common.h"
+
+#include <emscripten.h>
+#include <emscripten/val.h>
+
+namespace onnxruntime {
+
+class GraphViewer;
+class NodeArg;
+
+namespace logging {
+class Logger;
+}
+
+namespace webnn {
+
+bool GetShape(const NodeArg& node_arg, std::vector<int64_t>& shape, const logging::Logger& logger);
+
+bool IsInputSupported(const NodeArg& node_arg, const std::string& parent_name, const logging::Logger& logger);
+
+// Get a list of groups of supported nodes, each group represents a subgraph supported by WebNN EP
+std::vector<std::vector<NodeIndex>> GetSupportedNodes(const GraphViewer& graph_viewer,
+                                                      const emscripten::val& wnn_builder_,
+                                                      const logging::Logger& logger);
+
+inline InlinedHashMap<std::string, std::vector<std::string>> op_dependency = {
+    {"GRU", {"Split"}}};
+
+inline InlinedHashMap<std::string, std::string> op_map = {
+    {"Add", "add"},
+    {"Relu", "relu"},
+    {"LeakyRelu", "leakyRelu"},
+    {"Sigmoid", "sigmoid"},
+    {"Tanh", "tanh"},
+    {"BatchNormalization", "batchNormalization"},
+    {"Clip", "clamp"},
+    {"Conv", "conv2d"},
+    {"ConvTranspose", "convTranspose2d"},
```

Comment:
ditto.

Reply:
Since Bin is implementing convTranspose2d and will be landed soon. How about leave it as is?